src/config/dynamic-theme-fixes.config

```diff
@@ -2827,6 +2827,7 @@ img[src*="img-prod-cms-rt-microsoft-com.akamaized.net/cms/api/am/imageFileData/R
 img[src*="img-prod-cms-rt-microsoft-com.akamaized.net/cms/api/am/imageFileData/RE4pxBu"]
 img[src*="img-prod-cms-rt-microsoft-com.akamaized.net/cms/api/am/imageFileData/RE4pkvE"]
 img[src*="/static/images/inreplyto.svg"]
+img[src*="img-prod-cms-rt-microsoft-com.akamaized.net/cms/api/am/imageFileData/RE1Mu3b"]
```

Comment:
Change it to `.c-logo .c-image`

and 
```
CSS
.c-logo .c-image {
    background: transparent !important;
}
```

Reply:
For fixing image on light mode use, **white** instead of **${black}**. 
Inverting the logo isn't a good idea since it makes **Microsoft** hard to read and colours are changed drastically.

Reply:
Or we use the new Variable #2767 That I specially made to get the correct background/foreground as the ${} don't get parsed correctly due how it should work :/

Reply:
But if it's a solid image how does changing the background fix it?

Reply:
I will add then @Gusted